tracing.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+from opentelemetry import trace
+from opentelemetry.exporter.otlp.proto.http.trace_exporter import OTLPSpanExporter
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import BatchSpanProcessor, ConsoleSpanExporter
+
+
+def get_provider():
+    provider = TracerProvider()
```

Comment:
I just landed [a PR](https://github.com/opensafely-core/job-runner/pull/717) to jobrunner to use a top level Resource on the provider to add some context to every span. Might be worth doing the same here.